docs/how-tos/sensor-data-query-sdk.md

```diff
@@ -145,6 +158,8 @@ if __name__ == '__main__':
     asyncio.run(main())
 ```
 
+Make sure to replace the value in line 29 with your correct sensor name, line 35 with your organization ID which you can get by running `viam organizations list`, and line 37 with your location ID which you can get by running `viam locations list`.
```

Comment:
This probably isn't visible enough. I think make the `<location-id>` all caps `<LOCATION-ID>` and add a
```
# TODO: Replace  <LOCATION-ID> with your location ID
```
above it (same or org id).

And then move this explainer above the code snippet into a note. That way it'll definitely be seen.

You could also pull the strings into variable before the query is called. That might also make it more visible. And you could even add error proofing:
```python
LOCATION_ID = ""
ORG_ID = ""
if not LOCATION_ID or not ORG_ID:
    print("Please specify your location ID and organization ID and rerun this script")
```

Reply:
you can also change the highlighting on the snippet above to call out those lines

Reply:
done